myning/objects/plant.py

```diff
@@ -64,19 +56,64 @@ def from_dict(cls, dict: dict):
 
         return cls
 
+    def sow(self):
```

Comment:
I moved `from_dict` up to be with the init because these two methods are implementations of the base classes `Item` and `Object`, but I don't really have strong opinions on it having to stay this way so I wouldn't mind moving them back if you feel it's better that way

Reply:
Ohh that wasn't obvious to me front the diff. Yeah that can stay here, but let's move the icon one back down

Reply:
done!